packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -372,4 +372,25 @@ export class TreeViewWidget extends TreeWidget {
             }
         }
     }
+
+    private _message: string | undefined;
+    get message(): string | undefined {
+        return this._message;
+    }
+
+    set message(message: string | undefined) {
+        this._message = message;
```

Comment:
pleas guard if message does not change to avoid unnecessary rendering

Reply:
OK.